### PR TITLE
Eth-connector: switch to near-plugins

### PR DESCRIPTION
**Pull request by olga24912**
Changing access control API to near-plugins, to be compatible with
https://github.com/aurora-is-near/aurora-eth-connector/pull/71

### CODE REVIEW

File: eth-connector/tests/fungible_token_tests.rs [hunk 1]
```diff
@@ -260,28 +258,77 @@ async fn test_storage_balance_bounds() {
...
-async fn test_set_paused_flags() {
+async fn test_pa_pause_feature() {
     let contract = deploy_and_init().await.unwrap();
     contract
-        .set_paused_flags(UNPAUSE_ALL)
+        .pa_pause_feature("withdraw".to_string())
         .max_gas()
         .transact()
         .await
         .unwrap();
```
Comment (aleksuss) [inline eth-connector/tests/fungible_token_tests.rs:268]: As I remember, to unwrap is not enough to check that everything is ok. The `ExecutionFinalResult` has methods `is_success` and `is_failure`. I would add an assertion that the result of the transaction is successful like this.
```rust
 let result = contract
         .pa_pause_feature("withdraw".to_string())
         .max_gas()
         .transact()
         .await
         .unwrap();
assert!(result.is_success());
```

**Review by aleksuss — APPROVED** [review]
Overall looks good. But there are no new lines in the `Cargo.toml` files 😄

File: eth-connector/tests/fungible_token_tests.rs (hunk 1 above)
Comment (olga24912) [reply eth-connector/tests/fungible_token_tests.rs:271]: fixed: https://github.com/aurora-is-near/aurora-workspace/pull/39/commits/c1ff238262c6783c33d1cd04a9b9b2a51f52469b

File: res/mock_eth_connector/Cargo.toml [hunk 2]
```diff
@@ -18,3 +19,6 @@ lto = true
 debug = false
 panic = "abort"
 overflow-checks = true
+
+[patch.crates-io]
+parity-secp256k1 = { git = 'https://github.com/paritytech/rust-secp256k1', rev = "d05fd8e" }
```
Comment (karim-en) [inline res/mock_eth_connector/Cargo.toml:24]:
```suggestion
parity-secp256k1 = { git = 'https://github.com/paritytech/rust-secp256k1', rev = "d05fd8e" }

```

**Review by karim-en — CHANGES_REQUESTED** [review]

**Review by karim-en — APPROVED** [review]

**Review by mrLSD — APPROVED** [review]

**Pull request merged**